cdap-docs/_common/common-build.sh

```diff
@@ -236,12 +225,6 @@ function make_zip() {
   zip -qr ${ZIP_DIR_NAME}.zip ${PROJECT_VERSION}/* --exclude .DS_Store
```

Comment:
What would you suggest as a replacement? ``tar.gz``? That might not be acceptable on Windows.